server/build.gradle

```diff
@@ -148,6 +148,7 @@ dependencies {
 
   // protobuf
   api "com.google.protobuf:protobuf-java:${versions.protobuf}"
+  implementation "org.jboss.spec.javax.annotation:jboss-annotations-api_1.2_spec:${versions.jboss_annotation}"
```

Comment:
I picked jboss because it is already being used:

https://github.com/opensearch-project/OpenSearch/blob/277eb3d59fb9edcae8575577ec135032e3563f2e/qa/wildfly/build.gradle#L44

But I can switch.  I may need to generate a new LICENSE file for this.  Let me check.

Reply:
@reta 'jakarta.annotation' has 'jakarta.annotation.Generated'.  The one used by protobuf is 'javax.annotation.Generated'.  My IDE recommends either 'jboss.annotation' or 'javax.annotation' (https://mvnrepository.com/artifact/javax.annotation/javax.annotation-api/1.3.2).  I chose the former simply because it was already in use in this project.

Reply:
Thanks @austintlee that's the right one (with `javax.*` still)